operator/v1/types_machineconfiguration.go

```diff
@@ -32,6 +32,15 @@ type MachineConfigurationSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
 
 	// TODO(jkyros): This is where we put our knobs and dials
+
+	// managedBootImages allows configuration for the management of boot images for machine
+	// resources within the cluster. This configuration allows users to select resources that should
+	// be updated to the latest boot images during cluster upgrades, ensuring that new machines
+	// always boot with the current cluster version's boot image. When omitted, no boot images
+	// will be updated.
+	// +openshift:enable:FeatureSets=TechPreviewNoUpgrade
```

Comment:
Need a `CustomNoUpgrade` on this too else you'll break anyone who tries to use that featureset, see other uses of this tag